src/components/Sidebar/styles.ts

```diff
@@ -0,0 +1,29 @@
+
+import { makeStyles } from '@mui/styles';
+
+export default makeStyles(() => ({
+  sideBar: {
+    height: '100vh',
+    backgroundColor: '#3440e3',
+  },
+  imageLink: {
+    display: 'flex',
+    justifyContent: 'center',
+    padding: '10% 0',
+  },
+  image: {
+    width: '70%',
+  },
+  subHeader: {
+    backgroundColor: '#3440e3 !important',
+    color: '#a5a6c2 !important',
+  },
+  links: {
+    color: 'white',
+    textDecoration: 'none',
+
+  },
+  mainImages: {
```

Comment:
Empty class?

Reply:
Fixed - was meant to reduce the size of the menu item images, It only needed a change of height so I have done an inline styling for that property alone